mesa/space.py

```diff
@@ -512,10 +513,11 @@ def position_agent(
         if x == "random" or y == "random":
             if len(self.empties) == 0:
                 raise Exception("ERROR: Grid full")
-            coords = agent.random.choice(sorted(self.empties))
+            num_occupied = self.width * self.height - len(self.empties)
```

Comment:
The docstring says `Use 'move_to_empty()' when you want agents to jump to an empty cell.`, which is outdated.

Reply:
We should say that setting either x or y to "random" gives the same behavior as `move_to_empty`.

Reply:
I'm going to fix it.

Reply:
By the way, is `swap_pos` not implemented? I can't find it.

Reply:
[6e513de](https://github.com/projectmesa/mesa/pull/1457/commits/6e513de2f3b047b7e878fede95f88a2e24fe826b) - The docstring updated.